roles/keystone/tasks/monitoring.yml

```diff
@@ -17,6 +17,19 @@
                args="--service keystone --criticality {{ keystone.monitoring.sensu_checks.check_keystone_api.criticality }}"
   notify: restart sensu-client
 
+- block:
+  - name: create directory for below symbolic link
+    file:
+      path: /opt/openstack/current/
+      state: directory
+  
+  - name: add symolic link for keystone expired token check (rhel distro only)
+    file: 
+      src: /
```

Comment:
it works, but no so good.  now I changed the symbolic link from /bin/python->/opt/openstack/current/keystone/bin/python. please review again